Day6/processData.js

```diff
@@ -0,0 +1,29 @@
+function processData(input) {
+  //Enter your code here
+  message = input.split("\n");
```

Comment:
The message variable here is missing `datatype` which causes syntax error, you should either choose to go with `let` or `const` depending on the changing behavior of the variable